test/Driver/coverage-prefix-map.swift

```diff
@@ -3,7 +3,7 @@
 // RUN: %target-swiftc_driver -### -coverage-prefix-map old=n=ew %s 2>&1 | %FileCheck %s -check-prefix CHECK-COMPLEX
 // RUN: %target-swiftc_driver -### -coverage-prefix-map old= %s 2>&1 | %FileCheck %s -check-prefix CHECK-EMPTY
 
-// CHECK-INVALID: error: invalid argument 'old' to -coverage-prefix-map
+// CHECK-INVALID: error: values for '-coverage-prefix-map' must be in the format 'original=remapped', not 'old'
```

Comment:
I may be nitpicking, but this phrasing is a little harder for me to parse than the original because I interpret it as saying that the value the user provided is being treated as a contradictory format specification (i.e., "must be in the format 'this', not _(the format)_ 'that'"). It's subtle, but there's a slight disconnect between the first one being a format specification and the second being the actual value, while giving them both the same grammatical weight/role in the message.

If it's preferred to keep the form of the message that places the description of the correct behavior first, then how about this tweak to add a little bit more clarity?

`values for '-{coverage,debug}-prefix-map' must be in the format 'original=remapped', but 'old' was provided`

Reply:
Makes sense to me! I adopted your suggested wording